sdk/src/lib.rs

```diff
@@ -1,30 +1,48 @@
-pub mod account;
-pub mod account_utils;
-pub mod bpf_loader;
-pub mod client;
-pub mod fee_calculator;
-pub mod genesis_block;
-pub mod hash;
-pub mod inflation;
-pub mod instruction;
-pub mod instruction_processor_utils;
-pub mod loader_instruction;
-pub mod message;
-pub mod native_loader;
-pub mod packet;
-pub mod poh_config;
+#[macro_use]
+extern crate cfg_if;
+
 pub mod pubkey;
-pub mod rent;
-pub mod rpc_port;
-pub mod short_vec;
-pub mod signature;
-pub mod system_instruction;
-pub mod system_program;
-pub mod system_transaction;
-pub mod sysvar;
-pub mod timing;
-pub mod transaction;
-pub mod transport;
+
+// On-chain program modules
+cfg_if! {
```

Comment:
Do we run into issues if we also include these three modules in the kitchen sink?  Just wondering

Reply:
I think those three modules could be built along with the kitchen sink.  They are program-specific, do you have a reason to do so?

Reply:
I don't have a strong use case, loosely thinking that it could be useful to have access in a kitchen sink unit test.  But nbd right now, we can do that later if needed